package.json

```diff
@@ -35,26 +35,29 @@
     "@creditkarma/thrift-server-core": "^0.16.1",
     "@emotion/babel-preset-css-prop": "^10.1.0",
     "@emotion/core": "^10.0.35",
+    "@emotion/css": "^11.1.3",
+    "@emotion/react": "^11.1.5",
+    "@emotion/server": "^11.0.0",
     "@emotion/styled": "^10.0.27",
     "@guardian/apps-rendering-api-models": "^0.11.0",
-    "@guardian/atoms-rendering": "2.0.7",
+    "@guardian/atoms-rendering": "^12.0.0",
     "@guardian/bridget": "^1.8.0",
     "@guardian/content-api-models": "^15.10.0",
     "@guardian/content-atom-model": "^3.2.4",
     "@guardian/discussion-rendering": "^3.2.1",
     "@guardian/image-rendering": "github:guardian/image-rendering#semver:^5.0.2",
     "@guardian/node-riffraff-artifact": "^0.1.9",
     "@guardian/renditions": "git+https://github.com/guardian/renditions.git#0.1.1",
-    "@guardian/src-brand": "^2.6.0",
-    "@guardian/src-button": "^2.5.0",
-    "@guardian/src-ed-lines": "^2.6.0",
-    "@guardian/src-foundations": "^2.5.0",
-    "@guardian/src-icons": "^2.8.1",
-    "@guardian/src-link": "^2.7.1",
-    "@guardian/src-radio": "^2.5.0",
-    "@guardian/src-svgs": "^2.8.1",
-    "@guardian/src-text-area": "^2.8.1",
-    "@guardian/src-text-input": "^2.7.1",
+    "@guardian/src-brand": "^3.3.0",
+    "@guardian/src-button": "^3.3.0",
+    "@guardian/src-ed-lines": "^3.3.0",
+    "@guardian/src-foundations": "^3.3.0",
+    "@guardian/src-icons": "^3.3.0",
+    "@guardian/src-link": "^3.3.0",
+    "@guardian/src-radio": "^3.3.0",
+    "@guardian/src-svgs": "^2.8.2",
```

Comment:
I recommend replacing the deprecated `@guardian/src-svgs` with `@guardian/src-icons` – see [Source v3.0.0 migration guide](https://docs.google.com/document/d/1qmHSCCCRs98LI9vN2B8I0yjqBAsRfJclEqcPma0wqbI/edit#heading=h.xva03f88jbdx)

Reply:
Thanks @SiAdcock will do 👍